sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.aest.controller.assessOfferingChangeRequest.e2e-spec.ts

```diff
@@ -305,6 +310,119 @@ describe("EducationProgramOfferingAESTController(e2e)-assessOfferingChangeReques
     },
   );
 
+  it(
+    "Should determine the offering change for an offering with one or more applications not in completed status " +
```

Comment:
can the description be like

`Should approve the offering change when the offering has one or more applications not in Completed status and add a institution note on approval.`

Reply:
Sure. The description is updated.